### ISSUE TITLE
Define how throwaway accounts are to be created

### ISSUE THREAD

**Issue by cuongdo** [OriginProtocol/origin-js#509]
To simplify deployment of contracts, we will:
1. Deploy contracts using a "throwaway account"
2. Use the token CLI to set the owners of the token and marketplace contracts to a multi-sig wallet

The question is: what are the steps to create the "throwaway account"? Also, the throwaway account needs ETH to pay for the gas for the deployments. So, this process needs to include transferring enough ETH to the throwaway account. Plus, we should transfer back any ETH before we throw away the account.

cc @joshfraser

**Comment by joshfraser** [comment]
It's easy to create a new wallet in Metamask. I'm happy to send funds to whoever is doing the deploy.

**Comment by franckc** [comment]
We currently deploy *all* our contracts (token, marketplace, identity) using the *same* account because we use the truffle migrate tool.

Are we comfortable throwing away the account we use for deploying the [identity contracts](https://github.com/OriginProtocol/origin-js/tree/master/contracts/contracts/identity) ?
Is there any chance we would need that account for some operations on those contracts in the future ? I can't think of anything but CCing @tyleryasaka and @nick to make sure...

**Comment by joshfraser** [comment]
We don't actually have to throw away the keys, the point is that the ownership is going to be transferred to a mutisig wallet. The identity contacts don't have an owner, so it shouldn't matter for them.

**Comment by franckc** [comment]
Yes, you are right, the identity contracts don't have an owner so should not matter.
